src/__tests__/setup.ts

```diff
@@ -0,0 +1,5 @@
+import { createSerializer } from './kuma-serializer';
+
+export function setup() {
+  expect.addSnapshotSerializer(createSerializer());
+}
```

Comment:
[HACK]
When use `setupFiles` for run this, the bug which the styles string inside dom snapshot is duplicated occurs.
So, for now, this `setup` is directly imported from each test files, and used in `beforeAll`